crypto/x509/x509spki.c

```diff
@@ -131,3 +132,45 @@ char *NETSCAPE_SPKI_b64_encode(NETSCAPE_SPKI *spki) {
   OPENSSL_free(der_spki);
   return b64_str;
 }
+
+int NETSCAPE_SPKI_print(BIO *out, NETSCAPE_SPKI *spki) {
+  if (out == NULL || spki == NULL || spki->spkac->pubkey == NULL ||
```

Comment:
I think this should be checking `spki->spkac == NULL` instead of `spki->spkac->pubkey`.